scripts/mypy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+echo 'Starting mypy checks'
+mypy allennlp --ignore-missing-imports
```

Comment:
that actually means "ignore libraries that we don't have type stubs for" (e.g. pytorch), otherwise it generates all sorts of type errors from the pytorch code itself